src/lib/components/02-components/menus/language-switcher/language-switcher.tsx

```diff
@@ -0,0 +1,77 @@
+import { useEffect, useState } from 'react';
+import SecondaryMenu, { MenuItemProps } from '../secondary-menu/secondary-menu';
+
+interface LanguageSwitcherProps {
+  activeLocale: AvailableLocales;
+  locales: AvailableLocales[];
+  url: string;
+}
+
+export type AvailableLocales =
+  | 'en'
+  | 'es'
+  | 'fr'
+  | 'de'
+  | 'it'
+  | 'ja'
+  | 'ko'
+  | 'pt-br'
+  | 'tr'
+  | 'zh-hans'
+  | 'zh-hant'
+  | 'ar';
+
+const matchLanguage: { [key in AvailableLocales]: string } = {
+  en: 'English',
+  es: 'Español (América Latina)',
+  fr: 'Français',
+  de: 'Deutsch',
+  it: 'Italiano',
+  ja: '日本語',
+  ko: '한국어',
+  'pt-br': 'Português',
+  tr: 'Türkçe',
+  'zh-hans': '简体中文',
+  'zh-hant': '繁體中文',
+  ar: 'العربية',
+};
+
+export default function LanguageSwitcher({
+  activeLocale,
+  locales,
+  url,
+}: LanguageSwitcherProps) {
+  const [items, setItems] = useState<MenuItemProps[]>([]);
```

Comment:
I'm pretty sure you don't need a state for this. Just pass the item list you process below directly to the `SecondaryMenu` component

Reply:
Done